microprofile.jdt/com.redhat.microprofile.jdt.core/src/main/java/com/redhat/microprofile/jdt/core/PropertiesManager.java

```diff
@@ -109,36 +111,104 @@ public MicroProfileProjectInfo getMicroProfileProjectInfo(IJavaProject javaProje
 		if (LOGGER.isLoggable(Level.INFO)) {
 			LOGGER.info("Start computing MicroProfile properties for '" + info.getProjectURI() + "' project.");
 		}
+		SubMonitor mainMonitor = SubMonitor.convert(monitor,
+				"Collecting MicroProfile properties for '" + javaProject.getProject().getName() + "' project", 100);

```

Comment:
The monitor is already called Microprofile properties collector, so to avoid redundancy, you can set the subtask as:
    
        Scanning properties for ...

Reply:
@fbricon updated